### DISCUSSION TITLE
GiEX GX02 Water valve - new ZigBee manufacturer ID

### DISCUSSION THREAD

**Discussion by mmirannalti** [Koenkk/zigbee2mqtt#23071] [Devices]
Hi everyone.
Sometime ago I have bought one of this water valve and it works great with Z2M.
So, I just bought a second one.
Apparently everything is the same but the paring is not successful.
I have noticed that the ZigBee manufacturer ID has slightly changed.

I am attaching the two screenshots, the old one and the new one.

Any idea how to fix this?

Thanks
Massimiliano 

![Screenshot_20240615-214044](https://github.com/Koenkk/zigbee2mqtt/assets/44399209/85479193-482b-4e9e-8540-13221a8daada)
![Screenshot_20240615-214001](https://github.com/Koenkk/zigbee2mqtt/assets/44399209/72190927-4244-4ffd-aa3e-b62f5b48dc70)

**Comment by mmirannalti** [comment]
I fixed the issue following this previous post instructions:
https://github.com/Koenkk/zigbee2mqtt/discussions/13487

Is it possible to have the same configuration linked to multiple fingerprints, rather than duplicating the whole js configuration file?

**Comment by Koenkk** [comment]
could you post the external definition that you use?

**Reply by mmirannalti** [reply]
Checking further, I saw the same issue was fixed with the following pull request:
https://github.com/Koenkk/zigbee-herdsman-converters/pull/7589

Is this already part of version 1.38.0 ?

**Reply by Koenkk** [reply]
No, but will be in the 1 July release

**Reply by mmirannalti** [reply]
Thanks.

BTW, you are doing an incredible work, I really appreciate it!

**Comment by mmirannalti** [comment]
As I was still having errors when trying to use the external converter approach, I have checked further and I have realized that the issue was already raised and fixed with a pull request.

Most probably it was not part of the stable version I am running so I have configured the EDGE version of the HA add-on nad now it works correctly.

Maybe one last comment: the combination model/manufacturer is recognized as QT06_2.
I bought mine as GX02 and it also looks like a GX02 (fluo green color).
Not so important as it works :-) 

![2024-06-17_10-22-34](https://github.com/Koenkk/zigbee2mqtt/assets/44399209/7d0aee24-a584-40ed-92b7-9d5efc4034f2)
![2024-06-17_10-21-37](https://github.com/Koenkk/zigbee2mqtt/assets/44399209/7f934f5f-55d9-4b9e-811c-40217ba6b4e4)
